### DISCUSSION TITLE
did something change? importing preact/compat

### DISCUSSION THREAD

**Discussion by TreetopFlyer** [esm-dev/esm.sh#618] [Q&A] [answered]
just `deno repl` and run 
`import React from "https://esm.sh/preact/compat"`

and you get 
```
Uncaught TypeError: Relative import path "preact" not prefixed with / or ./ or ../ and not in import map from "https://esm.sh/stable/preact@10.13.2/deno/hooks.js"
    at https://esm.sh/stable/preact@10.13.2/deno/hooks.js:2:25
    at async <anonymous>:2:31
```
what do? im working on a project and this used to run but now it doesn't.

**Comment by renhiyama** [comment]
It's been like 2 weeks since this issue is happening. Deno cant seem to use preact.

**Comment by renhiyama** [comment]
Please check whether there's an issue already or create one.

**Comment by ije** [comment, marked as answer]
@TreetopFlyer this has been fixed in v118, please try again, thanks!

**Reply by TreetopFlyer** [reply]
@ije you rock and esm is the best

**Reply by ije** [reply]
🤘
